app/client/src/utils/autocomplete/EntityDefinitions.ts

```diff
@@ -667,6 +667,11 @@ export const GLOBAL_FUNCTIONS = {
     "!doc": "Stop executing a setInterval with id",
     "!type": "fn(id: string) -> void",
   },
+  postMessageToTargetWindow: {
+    "!doc":
+      "Establish cross-origin communication between Window objects/page and iframes",
+    "!type": "fn(message: any, targetOrigin: any, transfer: [any])",
```

Comment:
```suggestion
    "!type": "fn(message: any, targetOrigin: string, transfer: [any])",
```
Shouldn't we have `targetOrigin` as a string?

Reply:
done - miss on my part